server/index.js

```diff
@@ -96,47 +119,79 @@ wss.on("connection", (sock, _) => {
        * }
        */
     } else if (data.messageType === "play") {
-      console.log("received play: " + JSON.stringify(data));
+      // console.log("received play: " + JSON.stringify(data));
       const room = rooms.get(data.roomId);
 
       if (!room) {
         console.error(`Invalid play. Room ${data.roomId} does not exist`);
+        return;
+      }
+
+      // If not your turn - reject play
+      if (room.currentTurn !== data.sender) {
```

Comment:
We could potentially use the socket id here to make sure that the socket that received the message is associated with the player whose turn it is. (this would be down the road - not right now), but it would prevent malicious players from saying they are the other person and playing for them.

Reply:
Ahh interesting. Didn't consider any socket identifiers. Is that on the `sock` var?

Reply:
Resolving for now post discussion - will stick with playerId for now